keypatch.py

```diff
@@ -1109,6 +1109,78 @@ def OnFormChange(self, fid):
         return 1
 
 
+class Binary_Fill_NOPs(idaapi.action_handler_t):
```

Comment:
the way you reuse this `Binary_Fill_NOPs` class for `Binary_Fill_NULLs` is a bit weird. i would do that differently: rename `Binary_Fill_NOPs` to `Binary_Fill`, and pass `fill_value` (either `0x90` or `0x00`) & label (either "Fill with 00's" or "Fill with NOPs") to the class constructor.